cli/src/commands/token/create/pub_ns.rs

```diff
@@ -15,7 +17,10 @@ pub struct Opts {
 impl Opts {
 	pub async fn execute(&self, ctx: &cli_core::Ctx) -> GlobalResult<()> {
 		let output = execute(ctx, self).await?;
+
 		print!("{}", output.token);
+		std::io::stdout().flush()?;
+		eprintln!();
```

Comment:
I might be mistaken here, but `print!` should print to `stdout` while `eprintln!` prints to `stderr`. Why is it fine to do this instead of just changing to a `println!` then flush?

Reply:
When CLI tools read a command's output, they typically only read stdout (assuming the command was successful. This is important because we are returning a token, so if we were to print the new line to stdout and the user put that value directly into an http request, the new line would cause an "invalid token" error. This approach makes it so that the stdout is still just the token, and the display to the console has a new line (required for zsh shells so the EOL mark doesn't print)